CRM/Utils/Hook/Joomla.php

```diff
@@ -74,7 +74,7 @@ public function invokeViaUF(
           $app = JCli::getInstance();
         }
         else {
-          $app = JApplicationCli::getInstance();
+          $app = Joomla\CMS\Factory::getApplication();
```

Comment:
Umm not necessary but would be better to have a leading slash, i.e. `\Joomla\CMS\Factory::getApplication();`

Reply:
@monishdeb updated that now